data/src/emulator.js

```diff
@@ -3610,39 +3610,41 @@ class EmulatorJS {
             const elem = this.createElement("div");
             let style = '';
             if (dpad.left) {
-                style += 'left:'+dpad.left+';';
+                style += 'left:' + dpad.left + ';';
             }
             if (dpad.right) {
-                style += 'right:'+dpad.right+';';
+                style += 'right:' + dpad.right + ';';
             }
             if (dpad.top) {
-                style += 'top:'+dpad.top+';';
+                style += 'top:' + dpad.top + ';';
             }
             elem.classList.add(controlSchemeCls);
             if (dpad.id) {
                 elem.classList.add(`b_${dpad.id}`);
             }
             elem.style = style;
             elems[dpad.location].appendChild(elem);
-            createDPad({container: elem, event: (up, down, left, right) => {
-                if (dpad.joystickInput) {
-                    if (up === 1) up=0x7fff;
-                    if (down === 1) down=0x7fff;
-                    if (left === 1) left=0x7fff;
-                    if (right === 1) right=0x7fff;
+            createDPad({
+                container: elem, event: (up, down, left, right) => {
```

Comment:
If we newline on the start of the object, we should newline after every property